pkg/ccl/backupccl/restore_online.go

```diff
@@ -74,10 +75,12 @@ func sendAddRemoteSSTs(
 	grp.GoCtx(func(ctx context.Context) error {
 		return genSpan(ctx, restoreSpanEntriesCh)
 	})
+	fromSystemTenant := isFromSystemTenant(dataToRestore.getTenantRekeys())
+	writeAtBatchTimestamp := writeAtBatchTS(ctx, dataToRestore.getSpans()[0], fromSystemTenant)
```

Comment:
hmmm, can you comment on why spans[0] is representative of all spans, e.g. what about during cluster restore of tenants and tables?

Reply:
i added a comment but it may be wrong. Do all target spans for a restore have the same mvcc compatibility?

A more specific question: will/should a cluster restore of a system tenant restore all of its tenants?

Reply:
I've plumbed writeAtBatchTiemstamp to the processing of each span, since we can't guarantee that all target spans   should have the same mvcc compatibility.

Reply:
> once we remove the include_all_virtual_clusters

👍